### ISSUE TITLE
Gracefully handle when device is disconnected

### ISSUE THREAD

**Issue by szszoke** [dagargo/overwitch#20]
Hello!

Overwitch should handle when a device is disconnected gracefully. Today it locks up and prints a lot of messages like this:

```
ERROR:jclient.c:214:(jclient_o2j): o2j: Unexpected frames with ratio -540517.047517 (output 0, expected 1024)
```

**Comment by szszoke** [comment]
I did some testing and I think it works great. There were no nasty sounds this time, just some error messages about the device possibly being disconnected and then Overwitch shut down.

**Comment by dagargo** [comment]
But I've just realized that Ctrl+C doesn't work. I'll take a look at it.

**Comment by dagargo** [comment]
`Ctrl+C` fixed in 9b1bbd80b5551e889e9552e0ee61a9007c163258.

I'm closing this as solved. Feel free to reopen this if needed.
